src/Core/NuGet/Package.nuspec

```diff
@@ -22,14 +22,18 @@
         <dependency id="Microsoft.Diagnostics.Tracing.EventSource.Redist" version="1.1.28"/>
       </group>
       <group targetFramework="net45">
+        <dependency id="System.Diagnostics.DiagnosticSource" version="4.4.0-beta-25124-07" />
       </group>
       <group targetFramework="net46">
+        <dependency id="System.Diagnostics.DiagnosticSource" version="4.4.0-beta-25124-07" />
       </group>
       <group targetFramework="wp8">
         <dependency id="Microsoft.Bcl.Async" version="1.0.168" />
+        <dependency id="System.Diagnostics.DiagnosticSource" version="4.4.0-beta-25124-07" />
```

Comment:
Not needed. We no longer build the wp8 flavor. It's just a binary copy of an older stable version.